build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -42,21 +50,24 @@ git clone https://github.com/iree-org/iree.git
 
 cd iree
 git submodule update --init
-cmake -GNinja -B ../iree-build/ -S . -DCMAKE_BUILD_TYPE=RelWithDebInfo -DIREE_ENABLE_ASSERTIONS=ON -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DIREE_ENABLE_LLD=ON -DIREE_HAL_DRIVER_CUDA=ON -DIREE_TARGET_BACKEND_CUDA=ON
+cmake -GNinja -B ../iree-build/ -S . -DCMAKE_CXX_FLAGS="-Wno-deprecated-builtins" -DCMAKE_BUILD_TYPE=RelWithDebInfo -DIREE_ENABLE_ASSERTIONS=ON -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DIREE_ENABLE_LLD=ON -DIREE_HAL_DRIVER_CUDA=ON -DIREE_TARGET_BACKEND_CUDA=ON
```

Comment:
What's the reason to add "-Wno-deprecated-builtins"?

Reply:
It would fail on warnings about using deprecated built-ins. It might be the version of clang being used on the GCP instance.

Reply:
This sounds a little bit weird because we don't observe this in CI (and my day-to-day workflow). Can you post the environment? (Like the OS and versions of clang).

Reply:
This is the thomas-a100-east gcp instance. OS is Debian 10 and clang-16.